app/__init__.py

```diff
@@ -134,6 +124,12 @@ def override_url_for():  # pylint: disable=unused-variable
         add_health_check(application, headers)
         start_dev_mode(application)
 
+    if settings.EQ_FLASK_DEBUG:
```

Comment:
I think we should rename the environment variable to `EQ_ENABLE_FLASK_DEBUG_TOOLBAR` and move the logic into the `start_dev_mode` method.